cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -331,6 +331,7 @@ void SbPlayerPipeline::Stop(const base::Closure& stop_cb) {
 
     LOG(INFO) << "Destroying SbPlayer.";
     player_bridge.reset();
+    video_stream_ = nullptr;
```

Comment:
1. |video_stream_| is managed by |demuxer_|, so we should set it to nullptr after |demuxer_->Stop()| is called.
2. We should also set |audio_stream_| to nullptr, though it's not used in any unguarded places now.
3. Now the code starts to rely "more" on the factor that `AppendStatisticsString()` is always called on |task_runner_|.  We should add a DCHECK() in it to ensure that.  I also briefly checked all usages of |audio_stream_| and |video_stream_| and they are somewhat safe (either guarded by some logic, or on |task_runner_|).  Please double check.

Reply:
Fixed.